gadgets/trace_oomkill/program.bpf.c

```diff
@@ -15,14 +16,9 @@
 
 struct event {
 	gadget_timestamp timestamp_raw;
-	gadget_mntns_id mntns_id;
-	__u32 fpid;
-	__u32 fuid;
-	__u32 fgid;
-	__u32 tpid;
+	struct gadget_process fprocess;
+	struct gadget_process tprocess;
```

Comment:
I wonder if it is worth to use the `gadget_process` for the tprocess which will only carry the pid, comm and mntsns. I'm not sure, also because having two mntnsid generate this warn:

```
multiple fields of "type:gadget_mntns_id" found in DataSource "oomkill", only "mntns_id" will be used
```

Additionally, please update the gadget.yml and documentation to show the columns that need to be shown.

Reply:
What about just using a u64 to hold the target mount namespace? (until we clarify what's the meaning of multiple mount namespaces on a structure)

Reply:
With the current PR the enrichment is done with `fprocess`, right? 

What I'm proposing is to use u64 for one of the namespaces to avoid the confusion.

Reply:
ok, now I get it. I think we should keep the same enrichment as on main. Let's ignore the warning for now, we can fix it later on.